File: dart/1005.dart
import 'dart:io';

void main() {
  double a = double.parse(stdin.readLineSync()!);
  double b = double.parse(stdin.readLineSync()!);
  double m = (a * 3.5 + b * 7.5) / 11.0;

  print('MEDIA = ${m.toStringAsFixed(5)}');
}

File: c/1005.c
#include <stdio.h>

int main()
{
    double n1;
    double n2;

    scanf("%lf", &n1);
    scanf("%lf", &n2);

    printf("MEDIA = %.5lf\n", (n1 * 3.5 + n2 * 7.5) / 11);
    return 0;
}
